src/main/java/it/gov/pagopa/payhub/activities/service/classifications/trclassifiers/TransferClassifier.java

```diff
@@ -32,7 +32,7 @@ public interface TransferClassifier {
 	 * @param transferDTO the transfer data.
 	 * @return the amount in cents, or {@code null} if the amount is not available.
 	 */
-	default Long getAmountCents(Transfer transferDTO) {
+	default Long getAmountCentsFromTransfer(Transfer transferDTO) {
```

Comment:
This is explicit, we could let ASIS
```suggestion
	default Long getAmountCents(Transfer transferDTO) {
```